packages/onboard-react/src/components/HardwareWallets/HardwareWallets.tsx

```diff
@@ -27,14 +26,12 @@ export const HardwareWallets: FC = () => {
       {!inProgress ? (
         <WalletList filter={FilterType.hardware} />
       ) : (
-        <div>
-          <button onClick={handleStepBack} className={styles.closeButton}>
-            <Icon icon={IconNames.ARROW_LEFT} size={12} />
-          </button>
+        <>
+          <ButtonBack label="Back to wallet menu" onClick={handleStepBack} />
```

Comment:
As Victor already mentioned, there is no label for the desktop version. However, for the sake of simplicity, I think you should talk to Gilad because it's not something that would distract me on a desktop either and maybe we could have it everywhere.

Also, if you'll talk to Gilad, ask him if the label should be "Back to wallet menu" everywhere because I think that once we're in the address selection step, it should say something like "Back to derivation path selection" because you're technically don't go to wallet menu directly.